Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1497,7 +1497,7 @@ public string GetProviderUrl(string account)
             },
             new AuthenticatedFeed()
             {
-                AccountUrlPattern = @"^https:\/\/pkgs.dev.azure.com\/(?<account>[a-zA-z0-9]+)\/",
+                AccountUrlPattern = @"^https:\/\/pkgs.dev.azure.com\/(?<account>[-a-zA-z0-9]+)\/",
```

Comment:
There's another AuthenticatedFeed in this list with a similar 'account' defined in it's regex. Can you fix that one too?

Reply:
Sure, updated.

Reply:
Do you plan on publishing a new release with this fix?

Reply:
I can make a new release once these changes are in. =)